src/applications/personalization/appointments/reducers/index.js

```diff
@@ -1,32 +1,27 @@
 import {
-  FETCH_APPOINTMENTS,
-  FETCH_APPOINTMENTS_SUCCESS,
-  FETCH_APPOINTMENTS_FAILURE,
-} from '../actions';
+  FETCH_CONFIRMED_FUTURE_APPOINTMENTS,
+  FETCH_CONFIRMED_FUTURE_APPOINTMENTS_FAILED,
+  FETCH_CONFIRMED_FUTURE_APPOINTMENTS_SUCCEEDED,
+} from '~/applications/personalization/dashboard-2/constants';
 
 const initialState = {
-  loading: false,
   data: [],
+  fetching: false,
 };
 
-export default function appointments(state = initialState, action) {
+export default (state = initialState, action) => {
   switch (action.type) {
-    case FETCH_APPOINTMENTS:
-      return {
-        ...state,
-        loading: true,
-      };
-    case FETCH_APPOINTMENTS_SUCCESS:
-      return Object.assign({}, state, {
-        data: action.data,
-        loading: false,
-      });
-    case FETCH_APPOINTMENTS_FAILURE:
-      return Object.assign({}, state, {
-        error: action.error,
-        loading: false,
-      });
-    default:
-      return state;
+    case FETCH_CONFIRMED_FUTURE_APPOINTMENTS: {
+      return { ...state, fetching: true };
```

Comment:
Just wanted to point out that when we do start storing appointment fetch errors in redux, we'll want to clean the error here. And might want to clear the `data` as well, either here or in the FAILED case.